### ISSUE TITLE
Add documentation website to improve discoverability

### ISSUE THREAD

**Issue by aviggiano** [OpenZeppelin/openzeppelin-community-contracts#20]
It would be nice to see what's in here just from reading the README

**Comment by aviggiano** [comment]
I agree that it is useful to keep in sync, although very useful. Maybe a documentation website like the one from the original contracts repo would be nice some day.

**Comment by ernestognw** [comment]
Good point. I think a documentation website is a good middle ground so I'll work on that over the following weeks.

I'm renaming the issue accordingly. Thanks for your feedback @aviggiano!

**Comment by gonzaotc** [comment]
Docs live at https://docs.openzeppelin.com/community-contracts/0.0.1/
